samples/features/sql-big-data-cluster/deployment/private-aks/scripts/deploy-private-aks-udr.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+#Get Subscription ID and service principles as input. It is used as default for controller, SQL Server Master instance (sa account) and Knox.
+
+while true; do
+    read -s -p "Your Azure Subscription: " subscription
+    echo
+    read -s -p "Your service principles ID: " sp_id
+    echo
+    read -s -p "Your service principles Password: " sp_pwd
+
+done
+
+#Define a set of environment variables to be used in resource creations.
+
+export SUBID=$subscription
+
+export REGION_NAME=northeurope
+export RESOURCE_GROUP=private-bdc-rg
+export SUBNET_NAME=aks-subnet
+export VNET_NAME=bdc-vnet
+export AKS_NAME=bdcaksprivatecluster
+export FWNAME=bdcaksazfw
+export FWPUBIP=$FWNAME-ip
+export FWIPCONFIG_NAME=$FWNAME-config
+
+export FWROUTE_TABLE_NAME=bdcaks-rt
+export FWROUTE_NAME=bdcaksroute
+export FWROUTE_NAME_INTERNET=bdcaksrouteinet
+
+
```

Comment:
I think you need an extra step here to set the context to the sub id specified by the user

Reply:
ok